contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -359,9 +360,33 @@ mod tests {
         );
     }
 
+    const TEST_DATA_1: &str = "./testdata/airdrop_stage_1_test_data.json";
+    const TEST_DATA_2: &str = "./testdata/airdrop_stage_2_test_data.json";
+
+    #[derive(Deserialize, Debug)]
+    struct Encoded {
+        account: String,
+        amount: Uint128,
+        root: String,
+        proofs: Vec<String>,
+    }
+
+    fn read_test_data(path: &str) -> Encoded {
```

Comment:
There is a nice trick you can use...

Replace:

```rust
const TEST_DATA_1: &str = "./testdata/airdrop_stage_1_test_data.json";
let test_data = read_test_data(TEST_DATA_1);
```

with

```rust
const TEST_DATA_1: &[u8] = include_bytes!("./testdata/airdrop_stage_1_test_data.json");
let test_data: Encoded = from_slice(TEST_DATA_1).unwrap();
```

Then you can remove `read_test_data`.